generators/app/templates/webapp/Component.ts

```diff
@@ -1,6 +1,6 @@
 import UIComponent from "sap/ui/core/UIComponent";
 import models from "./model/models";
-import { support } from "sap/ui/Device";
+import Device from "sap/ui/Device";
```

Comment:
This only works for UI5 >= 1.115.0:
https://github.com/ui5-community/generator-ui5-ts-app/blob/main/generators/app/templates/webapp/Component.ts#L4

Reply:
Thanks! I applied the same for the generic types but noticed this was only from 1.116 or higher or even 1.117?  (see my other comment) https://github.com/lemaiwo/generator-ui5-ts-app-fcl/blob/2fa2add5a4f39192e32c8253a20df4ec55b316ff/generators/app/templates/webapp/Component.ts#L99